lib/mito/geometry/CoordinateSystem.h

```diff
@@ -84,6 +106,10 @@ namespace mito::geometry {
             return 0.5 * (coordinates(point_a) + coordinates(point_b));
         }
 
+        // support for ranged for loops
+        inline auto begin() const { return std::cbegin(_coordinates_map); }
+        inline auto end() const { return std::cend(_coordinates_map); }
```

Comment:
The overload for {begin} mixes with {std::cbegin} (same for {end}). Shouldn't this be
```
inline auto begin() const { return std::begin(_coordinates_map); }
inline auto end() const { return std::end(_coordinates_map); }
```